README.md

```diff
@@ -1,12 +1,30 @@
 # todo
-Example project meant to demonstrate a production-ready, enterprise-style REST API using go-kit. 
+[![MIT](https://img.shields.io/github/license/jarri-abidi/todo)](https://github.com/jarri-abidi/todo/blob/master/LICENSE)
 
-To run:
+**todo** is an example application written in [Go](https://go.dev/) meant to demonstrate a production-ready, enterprise-style REST API using go-kit. 
+
+## Usage
+### Run
 ```
 docker-compose build && docker-compose up -d
 ```
-
-To stop:
+### Stop
 ```
 docker-compose down
-```
\ No newline at end of file
+```
+
+## How to contribute?
+Want to contribute? Awesome! The most basic way to show your support is to star the project or to raise issues. Pull requests are highly welcome.
+
+Please make sure to update tests as appropriate.
+
+## Maintainers
+[Jarri Abidi](https://github.com/jarri-abidi)\
+[Faisal Nisar](https://github.com/markhaur)
+
+## Credits
+This project exists, thanks to all the people who contribute.
+
+<a href="https://github.com/jarri-abidi/todo/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=jarri-abidi/todo" />
+</a>
```

Comment:
I think I'll write something about contributing guidelines myself when I'm ready. For now let's remove this.
Also, I don't want this project to be about contributors. Only the code is important.
```suggestion
```
```